src/api/cvc4cppkind.h

```diff
@@ -0,0 +1,2328 @@
+   *   -[2]: Term of sort String (string s2)
+   * Create with:
+   *   mkTerm(Kind kind, Term child1, Term child2)
+   *   mkTerm(Kind kind, const std::vector<Term>& children)
+   */
+  STRING_PREFIX,
+  /**
+   * String suffix-of.
+   * Checks whether a string s1 is a suffix of string 2. If string s1 is empty,
+   * this operator returns true.
+   * Parameters: 2
+   *   -[1]: Term of sort String (string s1)
+   *   -[2]: Term of sort String (string s2)
+   * Create with:
+   *   mkTerm(Kind kind, Term child1, Term child2)
+   *   mkTerm(Kind kind, const std::vector<Term>& children)
+   */
+  STRING_SUFFIX,
+  /**
+   * Integer to string.
+   * If the integer is negative this operator returns the empty string.
+   * Parameters: 1
+   *   -[1]: Term of sort Integer
+   * Create with:
+   *   mkTerm(Kind kind, Term child)
+   */
+  STRING_ITOS,
+  /**
+   * String to integer (total function).
+   * If the string does not contain an integer or the integer is negative, the
+   * operator returns -1.
+   * Parameters: 1
+   *   -[1]: Term of sort String
+   * Create with:
+   *   mkTerm(Kind kind, Term child)
+   */
+  STRING_STOI,
+  /**
+   * Constant string.
+   * Parameters:
+   *   See mkString()
+   * Create with:
+   *   mkString(const char* s)
+   *   mkString(const std::string& s)
+   *   mkString(const unsigned char c)
+   *   mkString(const std::vector<unsigned>& s)
+   *   mkConst(Kind kind, const char* s)
+   *   mkConst(Kind kind, const std::string& s)
+   */
+  CONST_STRING,
+  /**
+   * Conversion from string to regexp.
+   * Parameters: 1
+   *   -[1]: Term of sort String
+   * Create with:
+   *   mkTerm(Kind kind, Term child)
+   */
+  STRING_TO_REGEXP,
+  /**
+   * Regexp Concatenation .
+   * Parameters: 2
+   *   -[1]..[2]: Terms of Regexp sort
+   * Create with:
+   *   mkTerm(Kind kind, Term child1, Term child2)
+   *   mkTerm(Kind kind, const std::vector<Term>& children)
+   */
+  REGEXP_CONCAT,
+  /**
+   * Regexp union.
+   * Parameters: 2
+   *   -[1]..[2]: Terms of Regexp sort
+   * Create with:
+   *   mkTerm(Kind kind, Term child1, Term child2)
+   *   mkTerm(Kind kind, const std::vector<Term>& children)
+   */
+  REGEXP_UNION,
+  /**
+   * Regexp intersection.
+   * Parameters: 2
+   *   -[1]..[2]: Terms of Regexp sort
+   * Create with:
+   *   mkTerm(Kind kind, Term child1, Term child2)
+   *   mkTerm(Kind kind, const std::vector<Term>& children)
+   */
+  REGEXP_INTER,
+  /**
+   * Regexp *.
+   * Parameters: 1
+   *   -[1]: Term of sort Regexp
+   * Create with:
+   *   mkTerm(Kind kind, Term child)
+   */
+  REGEXP_STAR,
+  /**
+   * Regexp +.
+   * Parameters: 1
+   *   -[1]: Term of sort Regexp
+   * Create with:
+   *   mkTerm(Kind kind, Term child)
+   */
+  REGEXP_PLUS,
+  /**
+   * Regexp ?.
+   * Parameters: 1
+   *   -[1]: Term of sort Regexp
+   * Create with:
+   *   mkTerm(Kind kind, Term child)
+   */
+  REGEXP_OPT,
+  /**
+   * Regexp range.
+   * Parameters: 1
+   *   -[1]: Term of sort Regexp
```

Comment:
-[1]: lower bound character for the range
-[2]: upper bound character for the range.